### PR TITLE
Add install & run example instructions to the docs and the readme

### LINKED ISSUE #77: Add install & run example instructions to the docs and the readme
**context**
We need instruction to onboard more devs and users.

**ideas**
Don't reinvent the wheel and mimic the O.jl setup here.

**next steps**
do it.

### PR DESCRIPTION
**Pull request by szy21**
Closes #77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,4 +29,44 @@
...
+[source](https://github.com/CliMA/ClimaAtmos.jl) with:
+
+```
+$ git clone https://github.com/CliMA/ClimaAtmos.jl.git
+```
+
+Now change into the `ClimaAtmos.jl` directory with 
+
+```
+$ cd ClimateMachine.jl
```
Comment (bischtob) [inline README.md:46]: :) maybe not?

**Review by bischtob — CHANGES_REQUESTED** [review]
Just a few typos to correct. Otherwise this is good!

**Comment by bischtob** [comment]
Please also squash your commits and rebase on master so that we can merge it after the corrections :).

File: README.md (hunk 1 above)
Comment (szy21) [reply README.md:46]: 🤦‍♀️

**Comment by szy21** [comment]
@bischtob I just fixed the typos and rebased it. Thanks!

**Comment by bischtob** [comment]
Thank you @szy21. Great work.

**Review by bischtob — APPROVED** [review]
LGTM

**Pull request merged**